README.md

```diff
@@ -1,2 +1,28 @@
 # rclc
-ROS Client Library for the C language.
+
+A lightweight ROS 2 Client Library for the C language.
```

Comment:
@BorjaOuterelo Should we make a disclaimer that this repo is not really maintained as we suggest using rcl directly? Can/should we even remove it? (It's just a fork ...)

Reply:
@ralph-lange what is the new direction of this repository? As far as I understand It will have the micro-ROS extensions, right?